app/Main.hs

```diff
@@ -9,24 +12,35 @@ module Main (main) where
 import Control.Exception (throwIO)
 import Control.Monad
 import Data.Bool (bool)
+import qualified Data.HashMap.Strict as HashMap
 import Data.List (intercalate, sort)
 import Data.Maybe (fromMaybe, mapMaybe)
+import Data.Text (Text, pack, replace, unpack)
```

Comment:
Only import `Text`, for others use `T.pack`, etc. This is the style that is used in the rest of the codebase.

Reply:
Fixed!